src/amo/pages/Addon/index.js

```diff
@@ -503,6 +503,8 @@ export class AddonBase extends React.Component {
             )}
           </div>
 
+          <PermissionsCard version={currentVersion} />
```

Comment:
I believe we want this card to be below the `ContributeCard`.

And we want to move the `AddAddonToCollection` below `AddonMoreInfo`.

In other words, we want to swap `AddAddonToCollection` and `PermissionsCard`.

Reply:
Oh, I see. Fixed: 2d7e52d0e68920be6bbe69efd2d784f4d673da81